src/backendmodule/backend-module.component.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from "react";
```

Comment:
rename this to backend-modules-dependencies.component.tsx

Reply:
The encouraged and easier way to import react:

```suggestion
import React from "react";
```